### ISSUE TITLE
Background: transparent

### ISSUE THREAD

**Issue by juliomac** [davidjbradshaw/iframe-resizer#335]
I would like to have my iFrame using same background of parent page.

I have access to the host iframe and I have added the following CSS:

    body{
      background-color: transparent;
    }
It works fine as the page is opened. The parent background fits nicely.  However, when the iframe expands, it expands 'over' the parent page rather than expanding and moving down parent's page content.

I also tried using:

    <script>iFrameResize({
      log:true,
      heightCalculationMethod: 'lowestElement';
      bodyBackground: 'transparent';
    })
    </script>

But that does not seem to have any effect.

Is there any work around?

**Comment by davidjbradshaw** [comment]
That is not supported by browsers.

**Comment by juliomac** [comment]
I found it works, depending on the situation. 

Iframe resizes nicely with transparent background if iframe is not embedded inside a '''<table>''' . I have worked out a way to change iframe's body '''Style''' to transparent and/or "color" depending on which parent page is calling it from.

When "transparent" works fine, iframe "gets" its background color from the parent, when not, I assign a background color that best fits it ("white" in most cases).
